recipes/exparna/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: exparna
+  version: 1.0.1
+
+source:
+  git_url: https://github.com/BackofenLab/ExpaRNA.git
```

Comment:
please use https://github.com/BackofenLab/ExpaRNA/archive/1.0.1.tar.gz with checksum and test locally with our new simulate-travis.py file. See FAQs.md :)